### PR TITLE
Update Changelog

### LINKED ISSUE #221: Update `CHANGELOG.md`
For the last couple of PRs, additions to the `CHANGELOG.md` have been missed

### PR DESCRIPTION
**Pull request by danielfeismann**
resolves #221 

Depends on #244

### CODE REVIEW

**Comment by danielfeismann** [comment]
Unsure also to add changes in documentation. What do you thing?

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,21 +8,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Harmonized configuration of csv parameters [#149](https://github.com/ie3-institute/simona/issues/149)
 
+
 ### Fixed
 - Location of `vn_simona` test grid (was partially in Berlin and Dortmund)
 - Let `ParticipantAgent` die after failed registration with secondary services (prevents stuck simulation)
 - Fix default resolution of weather source wrapper [#78](https://github.com/ie3-institute/simona/issues/78)
+- Fix invalid thread allocation in GridAgent [#111](https://github.com/ie3-institute/simona/issues/111)
+- Fixed config auto-generation [#131](https://github.com/ie3-institute/simona/pull/131)
+- Fixed genConfigSample gradle task[#148(https://github.com/ie3-institute/simona/issues/148)
```
Comment (sebastian-peter) [inline CHANGELOG.md:40]: Broken link

Comment (sebastian-peter) [inline CHANGELOG.md:40]: Broken link

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,21 +8,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Re-organizing test resources into their respective packages [#105](https://github.com/ie3-institute/simona/issues/105)
 - BREAKING: Using snapshot version of PSDM
 - Simplified PrimaryServiceProxy due to changes in PSDM [#120](https://github.com/ie3-institute/simona/issues/120)
 - Improved handling of weights and their sum in determination of weather data [#173](https://github.com/ie3-institute/simona/issues/173)
 - Improving code readability in EvcsAgent by moving FreeLotsRequest to separate methods [#19](https://github.com/ie3-institute/simona/issues/19)
+- Ignore dependabot snapshot dependencies [#27](https://github.com/ie3-institute/simona/issues/27)
 - Sending termination message to external simulation on expected and unexpected shutdowns of SIMONA [#35](https://github.com/ie3-institute/simona/issues/35)
+- Change transformer calculation since changes in PSDM [#99](https://github.com/ie3-institute/simona/issues/99)
+- Adapt to changed PvInputModel of PSDM (elevationAngle) [#100](https://github.com/ie3-institute/simona/issues/100)
+- Re-organizing test resources to packages [#105](https://github.com/ie3-institute/simona/issues/105)
```
Comment (sebastian-peter) [inline CHANGELOG.md:27]: Is already mentioned (line 18)

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,21 +8,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Consolidate csv parameterization in config [#149](https://github.com/ie3-institute/simona/issues/149)
+- Change weather scheme to COSMO [PR#154](https://github.com/ie3-institute/simona/pull/154)
 - Improved implementation of `RefSystemParser` [#212](https://github.com/ie3-institute/simona/issues/212)
-- Removed Gradle task puml2png (Converting Plantuml to png / svg files) since it is no longer needed  [#230](https://github.com/ie3-institute/simona/issues/230)
 - Harmonized configuration of csv parameters [#149](https://github.com/ie3-institute/simona/issues/149)
 
+
 ### Fixed
 - Location of `vn_simona` test grid (was partially in Berlin and Dortmund)
```
Comment (sebastian-peter) [inline CHANGELOG.md:35]: Maybe add link [#72](https://github.com/ie3-institute/simona/issues/72)? :)

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,21 +8,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Let `ParticipantAgent` die after failed registration with secondary services (prevents stuck simulation)
```
Comment (sebastian-peter) [inline CHANGELOG.md:36]: [#76](https://github.com/ie3-institute/simona/issues/76)

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,21 +8,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Implement SQL source for primary data [#34](https://github.com/ie3-institute/simona/issues/34), [#101](https://github.com/ie3-institute/simona/issues/101)
+- Relevant scientific papers have been added to the documentation [#139](https://github.com/ie3-institute/simona/issues/139)
+- Add troubleshooting section to Users guide [#160](https://github.com/ie3-institute/simona/issues/160)
+- Adapt documentation to changed simonaAPI [#191](https://github.com/ie3-institute/simona/issues/191)
+- Implementing a new plugin framework for external simulations [#195](https://github.com/ie3-institute/simona/issues/195)
```
Comment (sebastian-peter) [inline CHANGELOG.md:14]: Honestly, these belong more to the `Changed` section

**Review by sebastian-peter — CHANGES_REQUESTED** [review]
Looks great, thanks for the work. I just have some minor comments, and at least one issue is missing: https://github.com/ie3-institute/simona/pull/220

File: CHANGELOG.md (hunk 5 above)
Comment (danielfeismann) [reply CHANGELOG.md:14]: Thanks a lot for all your comments!

File: CHANGELOG.md [hunk 6]
```diff
@@ -8,21 +8,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Improved implementation of `RefSystemParser` [#212](https://github.com/ie3-institute/simona/issues/212)
-- Removed Gradle task puml2png (Converting Plantuml to png / svg files) since it is no longer needed  [#230](https://github.com/ie3-institute/simona/issues/230)
 - Harmonized configuration of csv parameters [#149](https://github.com/ie3-institute/simona/issues/149)
 
 ### Fixed
 - Location of `vn_simona` test grid (was partially in Berlin and Dortmund)
 - Let `ParticipantAgent` die after failed registration with secondary services (prevents stuck simulation)
+- Fix location of "vn_simona" test grid [#72](https://github.com/ie3-institute/simona/issues/72)
+- Participant agents die on failed registration [#76](https://github.com/ie3-institute/simona/issues/76)
```
Comment (sebastian-peter) [inline CHANGELOG.md:37]: Sorry, I meant the links I provided with my comments as additions to the first two points of the `Fixed` section

**Review by sebastian-peter — CHANGES_REQUESTED** [review]

**Comment by danielfeismann** [comment]
!test

**Review by sebastian-peter — APPROVED** [review]
LGTM, thanks!

**Review by t-ober — CHANGES_REQUESTED** [review]
Thanks for tackling. Found some more 🙃

File: CHANGELOG.md [hunk 7]
```diff
@@ -8,21 +8,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Let `ParticipantAgent` die after failed registration with secondary services (prevents stuck simulation)
+- Location of `vn_simona` test grid (was partially in Berlin and Dortmund) [#72](https://github.com/ie3-institute/simona/issues/72)
+- Let `ParticipantAgent` die after failed registration with secondary services (prevents stuck simulation) [#76](https://github.com/ie3-institute/simona/issues/76)
 - Fix default resolution of weather source wrapper [#78](https://github.com/ie3-institute/simona/issues/78)
+- Fix invalid thread allocation in GridAgent [#111](https://github.com/ie3-institute/simona/issues/111)
+- Fixed config auto-generation [#131](https://github.com/ie3-institute/simona/pull/131)
+- Fixed genConfigSample gradle task[#148](https://github.com/ie3-institute/simona/issues/148)
+- Fixed some unreachable code [#167](https://github.com/ie3-institute/simona/issues/167)
+- Fix treatment of non-InitializeTrigger triggers in initialization within SimScheduler [#237](https://github.com/ie3-institute/simona/issues/237)
+
```
[the remark this answers is not in the document]
Comment (danielfeismann) [reply CHANGELOG.md:48]: Was there but as PR 131, changed it to the issue number...

File: CHANGELOG.md [hunk 8]
```diff
@@ -8,21 +8,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Implementing a new plugin framework for external simulations [#195](https://github.com/ie3-institute/simona/issues/195)
 - Improved implementation of `RefSystemParser` [#212](https://github.com/ie3-institute/simona/issues/212)
-- Removed Gradle task puml2png (Converting Plantuml to png / svg files) since it is no longer needed  [#230](https://github.com/ie3-institute/simona/issues/230)
 - Harmonized configuration of csv parameters [#149](https://github.com/ie3-institute/simona/issues/149)
+- Include missing images into Documentation [#151](https://github.com/ie3-institute/simona/issues/151)
+- Changing the export methode for diagrams [#156](https://github.com/ie3-institute/simona/issues/156)
+- Change references implementation in Documentation to bibtex [#174](https://github.com/ie3-institute/simona/issues/174) 
+- Update Model descriptions (Documentation) [#122](https://github.com/ie3-institute/simona/issues/122)
+- Changes of Getting Started Section (Documentation) [#124](https://github.com/ie3-institute/simona/issues/124) 
+- Update gradle [#176](https://github.com/ie3-institute/simona/issues/176)
```
[the remark this answers is not in the document]
Comment (danielfeismann) [reply CHANGELOG.md:38]: Thanks!

**Review by sebastian-peter — APPROVED** [review]

**Review by staudtMarius — APPROVED** [review]
Looks great. I found no problems with the issues I worked on.

**Review by t-ober — APPROVED** [review]

**Comment by danielfeismann** [comment]
!test

**Pull request merged**